src/bblocks_data_importers/who/ghed.py

```diff
@@ -0,0 +1,190 @@
+
+        Args:
+            raw_data: Raw data extracted from the GHED database. Use the `_extract_raw_data` method to get this data
+        """
+
+        cols = {
+            "country": "country_name",
+            "code": "iso3_code",
+            "variable name": "indicator_name",
+            "variable code": "indicator_code",
+            "Sources": "sources",
+            "Comments": "comments",
+            "Data type": "data_type",
+            "Methods of estimation": "methods_of_estimation",
+            "Country footnote": "country_footnote",
+        }
+
+        try:
+            return (
+                pd.read_excel(raw_data, sheet_name="Metadata")
+                .rename(columns=cols)
+                .loc[:, cols.values()]
+            )
+
+        except (ValueError, KeyError) as e:
+            raise DataFormatError(f"Error formatting metadata: {e}")
+
+    def _load_data(self):
+        """Load the data from the GHED database to the object"""
+
+        self._raw_data = self._extract_raw_data()
+        self._data = self._format_raw_data(self._raw_data)
+        self._metadata = self._format_metadata(self._raw_data)
+
+    def get_data(self) -> pd.DataFrame:
+        """Get the data from the GHED database
+
+        # TODO: Add functionality to filter the data by country, indicator, year, etc.
+
+        Returns:
+            A DataFrame with the data from the GHED database
+        """
+
+        if self._data is None:
+            self._load_data()
+
+        return self._data
+
+    def get_metadata(self) -> pd.DataFrame:
+        """Get the metadata for the GHED database
+
+        Returns:
+            A DataFrame with the metadata for the GHED database including sources, footnotes, comments, etc.
+            for each indicator-country pair
+        """
+
+        if self._data is None:
+            self._load_data()
+
+        return self._metadata
+
+    def clear_cache(self) -> None:
+        """Clear the data cached in the importer"""
+
+        self._raw_data = None
+        self._data = None
+        self._metadata = None
+        logger.info("Cache cleared")
+
+    def download_data(self, path: str, file_name="ghed") -> None:
+        """Download the raw data to disk.
+
+        This method saves the raw data to disk in the specified path as an Excel file.
+
+        Args:
+            path: Path to the directory where the data will be saved
+            file_name: Name of the file to save the data
+        """
+
+        # Check if the directory exists
+        if not os.path.exists(path):
+            raise FileNotFoundError(
+                f"The directory {path} does not exist. Please provide a valid directory."
+            )
+
+        if self._raw_data is None:
+            self._load_data()
+
+        # TODO: Handle file already exists
+
+        with open(path + f"/{file_name}.xlsx", "wb") as file:
+            file.write(self._raw_data.getvalue())
```

Comment:
elsewhere you've managed cache without writing to disk (I think it was your package or the WEO stuff right?)

How come here you've decided to save?

Reply:
I believed we discussed this is just to add some functionality to save the raw data to disk. It isn't used for caching. The raw data response is a BytesIO object and the raw data in the source is an excel file. This just allows the user to download the raw data file if they want. I'm not entirely convinced we need this functionality